bindings/python/manifold3d.cpp

```diff
@@ -433,6 +433,24 @@ NB_MODULE(manifold3d, m) {
           nb::arg("crossSection"), nb::arg("circular_segments") = 0,
           nb::arg("revolve_degrees") = 360.0,
           manifold__revolve__cross_section__circular_segments__revolve_degrees)
+      .def_static(
+          "level_set",
+          [](const std::function<float(float, float, float)> &f,
+             std::vector<float> bounds, float edgeLength, float level = 0.0,
+             float precision = -1) {
+            // Same format as Manifold.bounding_box
+            Box bound = {glm::vec3(bounds[0], bounds[1], bounds[2]),
+                         glm::vec3(bounds[3], bounds[4], bounds[5])};
+
+            std::function<float(glm::vec3)> cppToPython = [&f](glm::vec3 v) {
+              return f(v.x, v.y, v.z);
+            };
+            return Manifold::LevelSet(cppToPython, bound, edgeLength, level,
+                                      precision, false);
+          },
+          nb::arg("f"), nb::arg("bounds"), nb::arg("edgeLength"),
+          nb::arg("level") = 0.0, nb::arg("precision") = -1,
+          manifold__level_set__sdf__bounds__edge_length__level__precision)
```

Comment:
@wrongbad I can't seem to get this doc string to be found - any idea what I'm doing wrong?

Reply:
I think you need `manifold__level_set__sdf__bounds__edge_length__level__precision__can_parallel`.

Reply:
I haven't looked in detail yet, but in general you can find the auto-generated header in the cmake build folder, and look for the exact var-name. If it's missing then the source file might not be white-listed by the script yet, or the comment formatting is different somehow.